lib/decanter/parser/json_parser.rb

```diff
@@ -0,0 +1,12 @@
+module Decanter
+  module Parser
+    class JsonParser < ValueParser
+
```

Comment:
Since `json` is not a native Ruby data type, there is no specific allowed types. Instead all values passed to this parser will be parsed by `JSON.parse(val)`, unless they are nil or blank strings.